src/components/ReportActionItem/TaskView.tsx

```diff
@@ -56,22 +54,22 @@ function TaskView({report}: TaskViewProps) {
         <View>
             <OfflineWithFeedback
                 shouldShowErrorMessages
-                errors={report.errorFields?.editTask ?? report.errorFields?.createTask}
-                onClose={() => Task.clearTaskErrors(report.reportID)}
+                errors={report?.errorFields?.editTask ?? report?.errorFields?.createTask}
+                onClose={() => clearTaskErrors(report?.reportID)}
                 errorRowStyles={styles.ph5}
             >
                 <Hoverable>
                     {(hovered) => (
                         <PressableWithSecondaryInteraction
-                            onPress={Session.checkIfActionIsAllowed((e) => {
-                                if (isDisableInteractive) {
+                            onPress={checkIfActionIsAllowed((e) => {
+                                if (isDisableInteractive || !report?.reportID) {
                                     return;
                                 }
                                 if (e && e.type === 'click') {
                                     (e.currentTarget as HTMLElement).blur();
                                 }
 
-                                Navigation.navigate(ROUTES.TASK_TITLE.getRoute(report.reportID, Navigation.getReportRHPActiveRoute()));
+                                Navigation.navigate(ROUTES.TASK_TITLE.getRoute(report?.reportID, Navigation.getReportRHPActiveRoute()));
```

Comment:
```suggestion
                                Navigation.navigate(ROUTES.TASK_TITLE.getRoute(report.reportID, Navigation.getReportRHPActiveRoute()));
```

Should work, no? if you added `|| !report?.reportID` to the early return.